go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/ipfs/go-ds-measure
 
 require (
-	github.com/ipfs/go-datastore v0.1.1
+	github.com/ipfs/go-datastore v0.1.2-0.20191117181321-b6c12d5ad99c
 	github.com/ipfs/go-metrics-interface v0.0.1
 )
+
+go 1.13
```

Comment:
Let's stick with go 1.12 for now.

Reply:
Sure, go keeps autoadding the latest for me. I'll fix it.